atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/ValidatingQueryRewritingKeyValueService.java

```diff
@@ -112,6 +113,15 @@ public void delete(TableReference tableRef, Multimap<Cell, Long> keys) {
     }
 
     @Override
+    public void deleteRange(TableReference tableRef, RangeRequest rangeRequest) {
+        if (!rangeRequest.getColumnNames().isEmpty()) {
```

Comment:
As mentioned in the test, I'm kinda interested in getting this validation everywhere since it's so cheap.  But on the other side this is wrapped on all client KVSs (as per TransactionManagers.create())